README.md

```diff
@@ -198,6 +198,12 @@ Or run it without zookeeper container:
 
     docker run marathon:{version} --master local --zk zk://localhost:2181/marathon
 
+On Mac you have to use a special address instead of `localhost`:
+
+    docker run --network host --hostname $(hostname) mesosphere/marathon:v1.6.335 \
+        --master zk://docker.for.mac.host.internal:2181/mesos \
+        --zk zk://docker.for.mac.host.internal:2181/marathon
```

Comment:
why not?   ` docker run --network host --hostname $(hostname) mesosphere/marathon:v1.6.335 --master $(hostname):5050 --zk zk://$(hostname):2181/marathon`
or
` docker run --network host --hostname $(hostname) mesosphere/marathon:v1.6.335 --master zk://$(hostname):2181/mesos --zk zk://$(hostname):2181/marathon`

Reply:
for running with local zk and mesos-local... this configuration has the following issues:
1. it registers with the master but goes inactive immediately
2. I can't hit marathon port 8080

Has this been tested?  or you confident this works? or is there missing information?

Reply:
Yes, I did test it. `--master $(hostname):5050` did not work on my machine. I think this has to do with the VM the Docker image runs in.